collision/label.go

```diff
@@ -3,6 +3,7 @@ package collision
 const (
 	// NilLabel is used internally for spaces that are otherwise not
 	// given labels.
+	// TODO V3: why is this exported
```

Comment:
Is there not a case where an implementer wants to check returned labels to se if they are NilLabel?

They could still  cast the label to perform the same check but that seems more annyoing.

Reply:
I'm not sure. I don't know if you should have the ability to ask if something is unlabeled. Worth spending more time on.